packages/components/src/DraftConfigurator/OptionGroup/index.js

```diff
@@ -66,13 +66,19 @@ const OptionGroup = (props) => {
 
   return (
     <>
-      {props.options.map((name) => {
+      {props.options.map((nameOrGroup) => {
         let output = []
-        if (typeof name === 'object') {
+        if (typeof nameOrGroup === 'object') {
+          let group = nameOrGroup
           // Subgroup
-          for (let subGroup of Object.keys(name)) {
+          for (let subGroup of Object.keys(group)) {
```

Comment:
Can be a `const` (yes, even in for loops :) Something I learned later in life)

Reply:
I tried a few iterations of this and had some trouble. Thanks for the advice.

Reply:
Done/